config/jobs/kubernetes-sigs/kubespray/kubespray-presubmits.yaml

```diff
@@ -0,0 +1,20 @@
+presubmits:
+  kubernetes-sigs/kubespray:
+  - name: pull-kubespray-yamllint
+    annotations:
+      testgrid-dashboards: sig-cluster-lifecycle-kubespray
+      testgrid-tab-name: yamllint
+    branches:
+    - master
+    always_run: false
+    skip_report: true
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
```

Comment:
the job does not appear to need these presets (docker-in-docker and google service accounts), so the labels section can be removed.

Reply:
Good catch, probably comes from a copy paste. I removed that in the last commit.